ios/Runner/Info.plist

```diff
@@ -10,18 +10,22 @@
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
+  <key>NSPhotoLibraryUsageDescription</key>
+  <string>We need to access your photo library so you can choose your own pictures when creating your Memos</string>
```

Comment:
why is this string repeated here and in en.lproj?

Reply:
This one is the fallback if the user is running in a locale that doesn't have a dedicated localization file